src/python/zquantum/core/estimator.py

```diff
@@ -22,10 +25,25 @@ def get_estimated_expectation_values(
             backend (QuantumBackend): the backend that will be used to run the circuit
             circuit (Circuit): the circuit that prepares the state.
             target_operator (List[SymbolicOperator]): List of target functions to be estimated.
+            n_samples (int): Number of measurements done on the unknown quantum state. 
+            epsilon (float): an error term.
+            delta (float): a confidence term.
 
         Returns:
             ExpectationValues: expectation values for each term in the target operator.
         """
+        estimator_name = type(self).__name__
+        self._ignore_parameter(estimator_name, "epsilon", epsilon)
+        self._ignore_parameter(estimator_name, "delta", delta)
+
+        if n_samples is not None:
+            save = backend.n_samples
```

Comment:
Not a very good variable name – I have no idea what it means.

Reply:
I'd throw a warning here (see other comments).

Reply:
The line reads as "save backend.n_samples" because it saves backend.n_sample :) it's as clear as you can get 😆  Oh well, I figured someone would leave a comment... I'll make the variable name clearer (rather than the line itself)

Reply:
I'll throw a warning too.